src/templates/fr/2019/chapters/resource-hints.html

```diff
@@ -0,0 +1,382 @@
+          </div>
+        </div>
+        <figcaption>Figure 1. Adoption des indices de ressources.</figcaption>
+      </figure>
+      <p>La popularité relative de <code>dns-prefetch</code> n'est pas surprenante ; c'est une API bien connue (elle est apparue pour la première fois en <a href="https://caniuse.com/#feat=link-rel-dns-prefetch">2009</a>), elle est prise en charge par tous les principaux navigateurs et c'est le moins "coûteux" de tous les indices de ressources. Parce que <code>dns-prefetch</code> n'effectue que des recherches DNS, il consomme très peu de données et il y a donc très peu d'inconvénients à l'utiliser. <code>dns-prefetch</code> est particulièrement utile dans les situations de latence élevée.</p>
+      <p>Cela étant dit, si un site n'a pas besoin de supporter IE11 et les versions inférieures, passer de <code>dns-prefetch</code> à <code>preconnect</code> est probablement une bonne idée. À une époque où le HTTPS est omniprésent, <code>preconnect</code> permet d'améliorer les performances tout en restant peu risqué. Notez que contrairement à <code>dns-prefetch</code>, <code>preconnect</code> initie non seulement la recherche DNS, mais aussi la poignée de main TCP et la négociation TLS. La <a href="https://knowledge.digicert.com/solution/SO16297.html">chaîne de certificats</a> est téléchargée pendant la négociation TLS et cela coûte généralement quelques kilo-octets.</p>
+      <p><code>prefetch</code> est utilisé par 3&nbsp;% des sites, ce qui en fait l'indices de ressources le moins utilisé. Cette faible utilisation peut s'expliquer par le fait que <code>prefetch</code> est utile pour améliorer le chargement des pages suivantes, plutôt que celui de la page actuelle. Ainsi, il sera négligé si un site se concentre uniquement sur l'amélioration de sa page d'accueil, ou sur les performances de la première page consultée.</p>
+      <figure id="fig-2">
+        <div class="table-wrap">
+          <div class="table-wrap-container">
+            <table>
+              <tbody>
+                <tr>
+                  <th>Resource Hint</th>
+                  <th>Indices de ressources par page&nbsp;:<br />Médiane</th>
+                  <th>Indices de ressources par page&nbsp;:<br />90e percentile</th>
+                </tr>
+                <tr>
+                  <td><code>dns-prefetch</code></td>
+                  <td>2</td>
+                  <td>8</td>
+                </tr>
+                <tr>
+                  <td><code>preload</code></td>
+                  <td>2</td>
+                  <td>4</td>
+                </tr>
+                <tr>
+                  <td><code>preconnect</code></td>
+                  <td>2</td>
+                  <td>8</td>
+                </tr>
+                <tr>
+                  <td><code>prefetch</code></td>
+                  <td>1</td>
+                  <td>3</td>
+                </tr>
+                <tr>
+                  <td><code>prerender</code> (deprecated)</td>
```

Comment:
```suggestion
                  <td><code>prerender</code> (déprécié)</td>
```

Reply:
This has come back as I incorrectly put it in the HTML instead of the MD and presume you regenerated the HTML. Apologies!